release-notes/VERSION

```diff
@@ -9,6 +9,7 @@ Versions: 3.x (for earlier see VERSION-2.x)
 
 #5032: Move `jackson-datatype-jsr310` module into `jackson-databind`
  (implemented by @pjfanning)
+#5032: Rename `JavaTimeModule` from `jackson-datatype-jsr310` module as `JavaTimeModule`
```

Comment:
Hmmh. I am not sure this make sense... note on #5032 is on line right above?

Reply:
Right I thought it would be good to record  `JavaTimeFeature` -> `DateTimeFeature` for later

Reply:
Made a bit of changes, there's no strong push on my side.
Feel free to close if seems unncessary!